src/components/VisBrev.tsx

```diff
@@ -13,16 +13,18 @@ const texts = {
 
 interface VarselBrevProps {
   document: DocumentComponentDto[];
+  buttonText?: string;
 }
 
-export const VisBrev = ({ document }: VarselBrevProps) => {
+export const VisBrev = ({ document, buttonText }: VarselBrevProps) => {
```

Comment:
Går evt an å spesifisere default value for `buttonText` her (`{ document, buttonText = texts.visBrev }: VarselBrevProps`)